docker_prod/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+services:
+  app:
+    image: archivesspace/archivesspace:4.0.0-RC1
+    container_name: archivesspace
+    restart: on-failure
+    ports:
+      - "8080:8080"
+      - "8081:8081"
+      - "8082:8082"
+      - "8089:8089"
+      - "8090:8090"
+    depends_on:
+      - db
+      - solr
+    environment:
+      APPCONFIG_DB_URL: "jdbc:mysql://db:3306/archivesspace?useUnicode=true&characterEncoding=UTF-8&user=as&password=as123&useSSL=false&allowPublicKeyRetrieval=true"
```

Comment:
This should interpolate envvars (`password=$PASSWORD`) or require it be set via an env file? (more in db comments)